packages/react-server/core/logging/server.js

```diff
@@ -68,7 +68,7 @@ function errorInterceptor (level, msg, meta) {
 // massage the error into a format suitable for logging
 function normalizeError (err) {
 	if (err instanceof Error) {
-		return _.pick({
+		return _.filter({
```

Comment:
This is... confusing to me. This changes the return value from an object w/ `message` and `stack` keys to an array, right? That seems weird, given that later in this method we return an object. 

I'm also interested to know when this stopped working, because I'm _pretty_ sure (though not 100%) that it worked at some point.